operate/quickstart/run_service.py

```diff
@@ -89,6 +89,8 @@
         "quickstart_beta_expert_12": 25,
         "quickstart_beta_expert_15_mech_marketplace": 25,
         "quickstart_beta_expert_16_mech_marketplace": 25,
+        "quickstart_beta_expert_17_mech_marketplace": 25,
+        "quickstart_beta_expert_18_mech_marketplace": 25,
```

Comment:
Are these agent ids? they can be read from the staking contract.

Reply:
those agent Ids are to filter the staking contracts that we should show to the user as an option to stake in the quickstart
for example, we should not give them an option to stake in the mech staking contract, if they are running trader quickstart